benchmark/domains/synthetic_3.py

```diff
@@ -0,0 +1,99 @@
+"""Synthetic dataset. Custom parabolic test with irrelevant parameters."""
```

Comment:
Is this the correct docstring? Iirc, this test was meant as a very easy hybrid test, I think the test with the irrelevant parameters was something else.